README.md

```diff
@@ -67,6 +67,50 @@ this property must be set to `/discounted-icecreams` as follows:
 accessibility-statement.service-path = "/discounted-icecreams"
 ```
 
+### Integrating with Tracking Consent
+
+The [TrackingConsentSnippet](src/main/twirl/uk/gov/hmrc/play/views/layouts/TrackingConsentSnippet.scala.html)
+component generates the HTML SCRIPT tags necessary to integrate with [tracking-consent-frontend](https//www.github.com/hmrc/tracking-consent-frontend)
+
+Before integrating, it is important to remove any hardcoded snippets relating to GTM, GA or Optimizely. Tracking consent
+manages the enabling of these third-party solutions based on the user's tracking preferences. If they are not removed
+there is a risk the user's tracking preferences will not be honoured.
```

Comment:
Maybe make a note about the `GTMSnippet.scala.html` and `OptimizelySnippet.scala.html`, that the new `TrackingConsentSnippet.scala.html` should be used instead of these?